config/routes.rb

```diff
@@ -3,9 +3,9 @@
 Rails.application.routes.draw do
   devise_for :admins, controllers: { sessions: 'admins/sessions' }
   devise_for :users, controllers: { registrations: 'users/registrations' }
-  resources :calculators, only: %i[show edit]
+  resources :calculators, only: %i[show edit update]
```

Comment:
`edit` and `update` should be removed from this place. User can't do changes for `calculator`